unitex/src/fr/umlv/unitex/config/Config.java

```diff
@@ -570,6 +570,8 @@ public static JFileChooser getExploreGraphOutputDialogBox() {
         exploreGraphOutputDialogBox = new JFileChooser();
         PersonalFileFilter txtFilter =new PersonalFileFilter("txt",
                                           "Text Files");
+        if (sentenceDialogBox == null)
```

Comment:
Hint:
> Braces are used with if, else, for, do and while statements, even when the body is empty or contains only a single statement.

Concerning code style: 

1) projects files: follow the code style of the project
2) new files: follow the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used)

Reply:
I'm confused because we can see both code styles in the same file.

Reply:
Concerning code style:
1. projects files: follow the code style of the project, with the exception of  the following recommendations of [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html):   4.1.1 (`Use of optional braces`) , 4.3 (`One statement per line`) and 4.4 (`Column limit: 100`) 
2. new files: follow the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used)

I will update the contribution guide later to include these recommendations